From 28e80c68221f4a2ce0d0b3ea68cb151bd28b95b5 Mon Sep 17 00:00:00 2001
From: Maruthi Bayyavarapu <maruthi.bayyavarapu@amd.com>
Date: Thu, 3 Aug 2017 14:18:55 -0400
Subject: [PATCH 1109/4131] ASoC: AMD: enable ACP3x drivers build

ACP3x drivers can be build by selecting necessary kernel config option.
The patch enables build support of the same.

Signed-off-by: Maruthi Bayyavarapu <maruthi.bayyavarapu@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 sound/soc/amd/raven/Makefile | 4 ++++
 1 file changed, 4 insertions(+)
 create mode 100644 sound/soc/amd/raven/Makefile

diff --git a/sound/soc/amd/raven/Makefile b/sound/soc/amd/raven/Makefile
new file mode 100644
index 0000000..8c03e6f
--- /dev/null
+++ b/sound/soc/amd/raven/Makefile
@@ -0,0 +1,4 @@
+snd-pci-acp3x-objs	:= pci-acp3x.o
+snd-acp3x-pcm-dma-objs	:= acp3x-pcm-dma.o
+obj-$(CONFIG_SND_SOC_AMD_ACP3x)	 += snd-pci-acp3x.o
+obj-$(CONFIG_SND_SOC_AMD_ACP3x)	 += snd-acp3x-pcm-dma.o
-- 
2.7.4

